lib/parliament/request/url_request.rb

```diff
@@ -1,7 +1,21 @@
 module Parliament
   module Request
+    # API request object, allowing the user to build a request to an API.
+    #
+    # @since 0.7.5
+    #
+    # @attr_reader [String] base_url the base url of our API. (expected: http://example.com - without the trailing slash).
+    # @attr_reader [Hash] headers the headers being sent in the request.
     class UrlRequest < Parliament::Request::BaseRequest
-      def initialize(base_url: nil, headers: nil, builder: nil)
+      # Creates a new instance of Parliament::Request::UrlRequest.
+      #
+      # @see Parliament::Request::BaseRequest#initialize.
+      #
+      # @param [String] base_url the base url of our api. (expected: http://example.com - without the trailing slash).
+      # @param [Hash] headers the headers being sent in the request.
```

Comment:
This would be a good point to add an @example? How do the headers affect the end request? How should you structure these?